isolate_test.go

```diff
@@ -18,11 +18,11 @@ import (
 
 func TestIsolateTermination(t *testing.T) {
 	t.Parallel()
-	iso, _ := v8go.NewIsolate()
+	iso := v8go.NewIsolate()
 	defer iso.Dispose()
-	ctx, _ := v8go.NewContext(iso)
+	ctx := v8go.NewContext(iso)
 	defer ctx.Close()
-	//	ctx2, _ := v8go.NewContext(iso)
+	//	ctx2 := v8go.NewContext(iso)
```

Comment:
Should this comment just be deleted?

Reply:
Sure, it doesn't seem like uncommenting it out uncovers a bug or anything, so I've pushed a commit removing it